### ISSUE TITLE
Add the option to select OneNote Inks and convert them to picture 

### ISSUE THREAD

**Issue by 1001QAdotNET** [stevencohn/OneMore#818]
If you use a pen in OneNote and you use the Ink feature you can create sketches and diagrams manually.

However these remain individual inks and it is difficult to move them together. 

Would it be possible to convert a group of selected ink drawings to a picture?

**Comment by stevencohn** [comment]
Yes. Can copy and paste as a new image or replace the original ink with a new image. If replaced, it would not be possible to restore the original ink.

**Comment by 1001QAdotNET** [comment]
not being able to edit is OK once the drawing is finished , thanks for adding this to the queue

**Comment by stevencohn** [comment]
Another caveat. This is not a screen capture. So if ink is overlaying other context, like text, that content will not be included in the resultant image. This will only capture the selected ink and render that as a new image. The image will be appended to the page because ink has an absolute position on the page, unrelated to the other content; it's impossible to determine its relative position to paragraphs, for example.

**Comment by 1001QAdotNET** [comment]
that is fine and indeed I would hate if the background is captured
this should be targeted toward sketching. I am also using MS Whiteboard and the pen has no pressure sensitivity there (among other problems).
The only advantage that MS Whiteboard has is that it has live synchronization between two users accessing the same page which is not allways the case in OneNote
Overall OneNote Ink is much better for drawing than MS Whiteboard -hence the need to capture the image corresponding to a  group of inks. If you consider that the image is a collapsile element while Ink is not you an probably guess that there is an obvious advantage in this conversion. Yes positioning that new image must be tricky but I would set it up to the coordinate in the group of inks that is closest to the top left corner of the canvas/page if thast is doable

**Comment by 1001QAdotNET** [comment]
Another way to approach this, if the background is wanted, (ex:annotated picture) would be do just do a sceen capture. Of course it would be nice to have that integrated with OneMore somehow (ShareX is a pretty good program -It would be nice if it can be integrated in both directions -there seems to be a bacdoor for that via OneNote APIs -see the custom menu in the picture below)
https://i.imgur.com/2MBT9Yf.png

**Comment by stevencohn** [comment]
I should have remembered this... you can select multiple ink shapes, copy them, and then from the Paste menu, paste as image.

Does that work?

**Comment by 1001QAdotNET** [comment]
Yeap :-)) I thought I tried that .... Not sure how I missed that too since I was playing with it when I asked thank you !!
